src/migrations/env.py

```diff
@@ -1,16 +1,15 @@
 import os
 import sys
-
 from logging.config import fileConfig
 
 from alembic import context
 from sqlalchemy import engine_from_config, pool
 
-sys.path.append(".")
-
 from src.config import config as app_config
 from src.models import db
 
+sys.path.append(".")
```

Comment:
I believe this has to be before the `src.*` imports otherwise it can't import the packages.